src/hmem_cuda_gdrcopy.c

```diff
@@ -292,6 +293,45 @@ void cuda_gdrcopy_from_dev(uint64_t handle, void *hostptr,
 			  GDRCOPY_FROM_DEVICE);
 }
 
+static ssize_t cuda_gdrcopy_iov_buf(uint64_t handle,
+                                    const struct iovec *iov, size_t iov_count,
+                                    uint64_t iov_offset, void *buf,
+                                    size_t size, bool to_device)
```

Comment:
ofi_copy_hmem_iov_buf() uses ```int dir``` instead of bool to_device.  We can align with that and have the callers pass in GDRCOPY_TO/FROM_DEVICE directly, which ends up making the code easier to read  (Seeing GDRCOPY_TO_DEVICE is more descriptive than seeing 'true')

Reply:
Good idea. Addressed.